wordpress/wordpress-api.ts

```diff
@@ -1,10 +1,23 @@
 
 /**
- * `GET & POST PAGES /wp/v2/pages`
+ * `GET PAGES /wp/v2/pages`
  */
 export const WORD_PRESS_API_GET_PAGE_PATH = '/wp-json/wp/v2/pages';
 
+
+/**
+ * `GET REFERENCES /wp/v2/references`
+ */
+export const WORD_PRESS_API_GET_REFERENCES_PATH = '/wp-json/wp/v3/references';
+
+
+/**
+ * `GET USER PROFILES /wp/v2/userprofiles`
+ */
+export const WORD_PRESS_API_GET_USER_PROFILES_PATH = '/wp-json/wp/v3/userprofiles';
+
+
 /**
- * `GET & POST POSTS /wp/v2/posts`
+ * `GET POSTS /wp/v2/posts`
```

Comment:
Same note about GET and POST to be their own contants -- even if the value is *now* same value, it's essentially different route.